test/internal/lib/connectors/services/water/BillingBatchService.js

```diff
@@ -114,4 +114,27 @@ experiment('services/water/BillingBatchService', () => {
       expect(url).to.equal(`https://example.com/water/1.0/billing/batches/${batchId}/account/${accountId}`);
     });
   });
+
+  experiment('.approveBatch', () => {
+    test('passes the expected URL to the service request', async () => {
+      const batchId = uuid();
+      await service.approveBatch(batchId);
+      const [url] = serviceRequest.post.lastCall.args;
+      expect(url).to.equal(`https://example.com/water/1.0/billing/batches/${batchId}/approve`);
+    });
+  });
+
+  experiment('.cancelBatch', () => {
+    test('passes the expected URL to the service request', async () => {
+      const batchId = uuid();
+      await service.cancelBatch(batchId);
+      const [url] = serviceRequest.delete.lastCall.args;
+      expect(url).to.equal(`https://example.com/water/1.0/billing/batches/${batchId}`);
+    });
+  });
 });
+
+/**
+ * POST /water/1.0/billing/batches/${validBatchId}/approve
+ * DELETE /water/1.0/billing/batches/${validBatchId}
+ */
```

Comment:
Did you mean to leave the above comments in?

Reply:
No. Thanks for this, I had left a trail of my thoughts here.